prod.yml

```diff
@@ -21,6 +21,11 @@ files:
     filter:
       - '**/*.js'
       - '**/*.node'
+fileAssociations:
+  ext: 'rep'
+  name: 'Starcraft Broodwar Replay'
```

Comment:
'Broodwar' is not a thing, the trademark is 'StarCraft: Brood War'. I would personally just simplify both the name and description here. From Electron's docs, it seems like the name should be a lot shorter, so probably just:

```yaml
name: 'REP'
description: 'StarCraft Replay File'
```